app/questionnaire/router.py

```diff
@@ -343,6 +343,11 @@ def _is_section_enabled(self, section: Mapping) -> bool:
             return True
 
         enabled = section["enabled"]
+
+        routing_path_block_ids = self._path_finder.get_when_rules_block_dependencies(
+            section["id"]
+        )
```

Comment:
We should pass this into the old engine as well. Not need to test.

Reply:
Ok added and also `current_location=None` since we do that in the new evaluator as well (for clarity?).